core/storage/config/gae_models.py

```diff
@@ -137,15 +139,17 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
         """Model doesn't contain any data directly corresponding to a user."""
         return dict(super(cls, cls).get_export_policy(), **{
             'rules': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'rule_schema_version': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            'rule_schema_version': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'is_feature_flag': base_models.EXPORT_POLICY.NOT_APPLICABLE
```

Comment:
Perhaps used_as_feature_flag to avoid starting with a verb.

Reply:
Will do that.